src/test/java/bo/usfx/exampletdd/WellTests.java

```diff
@@ -7,9 +7,9 @@ public class WellTests {
 
     @Test
     public void simpleTest() {
-        Assert.assertEquals("Fail!", Well.well(new String[]{"bad", "bad", "bad"}));
-        Assert.assertEquals("Publish!", Well.well(new String[] {"good", "bad", "bad", "bad", "bad"}));
-        Assert.assertEquals("I smell a series!", Well.well(new String[] {
-                "good", "bad", "bad", "bad", "bad", "good", "bad", "bad", "good"}));
+        Assert.assertEquals(Well.well(new String[]{"bad", "bad", "bad"}), "Fail!");
```

Comment:
This change is already in the main branch, please update your branch every time that you push new code.